en/drafts/originals/image-classification-neural-networks.md

```diff
@@ -25,150 +25,154 @@ doi: tbc
+Transfer learning expands on a model that has already been trained on a separate group of images. Teachable Machine relies on [MobileNet](https://perma.cc/8FWJ-NDXH) as the basis for its transfer learning. MobileNet is a lightweight neural network designed to run on small devices with low-latency. Its training times are relatively quick, and we can start with fewer images. Of course, MobileNet was not trained on the images that we are interested in so how exactly can we use it? This is where transfer learning kicks in.
 
-You can think of transfer learning as modifying the final layer of a preexisting model to discern our image's "features." At first, these features are mapped to the categories that MobileNet is trained on, but in transfer learning, we overwrite this mapping to reflect our own categories. Thus, we can rely on the earlier layers to do most of the heavy lifting while still having the benefit of using the final layers for classification.
+You can think of transfer learning as a process of modifying the final layer of a preexisting model to discern our images' "features." At first, these features are mapped to the categories that MobileNet was trained on, but through transfer learning, we can overwrite this mapping to reflect our own categories. Thus, we can rely on the earlier layers to do most of the heavy lifting &mdashdetecting basic features and shapes&mdash while still having the benefit of using the final layers to recognize specific objects and perform classification.
 
 On the Teachable Machine home page, go ahead and click the "Get Started" button. Then, click "Image Project" and select "Standard image model."
 
-Once we have selected that we are interested in image classification, we can upload the images for each class. You will find that you can either "Upload" images or use your webcam to create new ones. We will be uploading all the images for each of our categories to the training folder.
+Now, we can begin uploading image samples for each class. You will find that you can either "Upload" images or use your webcam to create new ones. We will be uploading all the images for each of our categories to the training folder.
 
 Under "Class 1", click "Choose images from your files, or drag & drop here." Select the "aircraft" folder from inside the dataset "training" folder and drag it into the Teachable Machine window. Click the pencil icon next to "Class 1" and change the name to "aircraft".
 
 Repeat this process for the other folders in the dataset. After the second time, you will need to click "+ Add a class" for each new folder.
 
-{% include figure.html filename="add_classes.png" caption="Figure 6. Adding classes to Google Teachable Machine." %}
+{% include figure.html filename="add_classes.png" alt="Visual description of figure image" caption="Figure 6. Adding classes to Google Teachable Machine." %}
```

Comment:
There is no Figure 4 or 5 ? 

We move directly from Figure 3 at l.65 to Figure 6 at l.123.

Are they missing? Or do we need to re-number and re-name remaining images?